pupil/public/javascripts/mtc-check.js

```diff
@@ -1,25 +1,24 @@
-/* global Touch DocumentTouch Event */
-
 (function (global) {
   'use strict'
-  var GOVUK = global.GOVUK || {}
-  var startTime
-  var pageInterval = 5  // seconds allowed per page
-  var preloadInterval = 2 // seconds for student preparation
-  var deadline
 
-  var registerInputs = []
+  var GOVUK = global.GOVUK || {}
+  var defaultQuestionTimeLimit = 5  // seconds allowed per page
```

Comment:
It's a little surprising to see these figures here.   Before this change these figures were stored in a single place, at the top of the mtc-check file.  Now they are stored in two config objects and this file, as well as being overridden by the database values.

Surely, if the default values are in the pupil config then they don't need to be replicated here as well?